setup.py

```diff
@@ -22,5 +22,5 @@
         "Operating System :: OS Independent",
         "Programming Language :: Python",
     ],
-    requires=["django"],
```

Comment:
* would be nice to break this out into one item per line
* Ideally this would have bounded dependency versions